mozdownload/__init__.py

```diff
@@ -2,7 +2,9 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from __future__ import absolute_import, print_function, unicode_literals
+from __future__ import absolute_import, unicode_literals
+
+__version__ = '1.19'
 
 from mozdownload import cli
```

Comment:
This had to be moved up because the imported `cli` doesn't know about it otherwise.